tests/acceptance/features/bootstrap/DataExporterContext.php

```diff
@@ -0,0 +1,264 @@
+		// Get all the contexts you need in this context
+		$this->featureContext = $environment->getContext('FeatureContext');
+		SetupHelper::init(
+			$this->featureContext->getAdminUsername(),
+			$this->featureContext->getAdminPassword(),
+			$this->featureContext->getBaseUrl(),
+			$this->featureContext->getOcPath()
+		);
+
+		$scenarioId = \bin2hex(\random_bytes(6));
+		$this->scenarioDir = self::path("{$this->outputDir}/$scenarioId");
+		$this->featureContext->mkDirOnServer($this->scenarioDir);
+	}
+
+	/**
+	 * @AfterScenario
+	 *
+	 * @return void
+	 */
+	public function deleteLastExport() {
+		$this->scenarioDir = null;
+		$this->lastExportBasePath = null;
+		$this->lastExportPath = null;
+		$this->lastExportUser = null;
+		$this->lastExportMetadataPath = null;
+	}
+
+	/**
+	 * @When user :user is exported to path :path using the occ command
+	 * @Given user :user has been exported to path :path using the occ command
+	 *
+	 * @param string $user
+	 * @param string $path
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function exportUserUsingTheCli($user, $path) {
+		$internalPath = self::path("{$this->scenarioDir}/$path");
+		$serverRoot = $this->featureContext->getServerRoot();
+		$this->featureContext->mkDirOnServer($internalPath);
+		$this->featureContext->runOcc(['instance:export:user', $user, self::path("$serverRoot/$internalPath")]);
+
+		$this->lastExportBasePath = $internalPath;
+		$this->lastExportPath = self::path("{$this->lastExportBasePath}/$user/");
+		$this->lastExportUser = $user;
+		$this->lastExportMetadataPath = "{$this->lastExportPath}metadata.json";
+	}
+
+	/**
+	 * @Then the last export should contain file :path
+	 *
+	 * @param string $path
+	 *
+	 * Checks whether a given file is present physically
+	 * and inside metadata
+	 *
+	 */
+	public function theLastExportContainsFile($path) {
+		self::assertPathContainsExport($this->lastExportPath);
+		$this->assertFilePhysicallyExistInLastExport($path, "File $path does not exist");
+		$this->assertFileExistsInLastExportMetadata($path);
+	}
+
+	/**
+	 * @Then the last export should contain file :path with content :content
+	 *
+	 * @param string $path
+	 * @param string $content
+	 *
+	 * Checks whether a given file is present physically
+	 * and inside metadata
+	 *
+	 */
+	public function theLastExportContainsFileWithContent($path, $content) {
+		$this->assertFileExistsInLastExportMetadata($path);
+		$this->assertFilePhysicallyExistInLastExportWithContent($path, $content);
+	}
+
+	/**
+	 * @When a user is imported from path :path using the occ command
+	 * @Given a user has been imported from path :path using the occ command
+	 *
+	 * @param string $path
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function importUserUsingTheCli($path) {
+		$importPath = self::path("$this->dataDir/$path");
+		$this->featureContext->runOcc(['instance:import:user', $importPath]);
+	}
+
+	private static function assertPathContainsExport($path) {
```

Comment:
Note: this method is looking in the local file system for the expected folders and files. That works if the Behat test runner script and the system-under-test share the same file system. For now, it is OK. But we might need to enhance the testing app to assist a bit more with returning information about folders that exist (or not) on the system-under-test.

Anyway, I think this method is not being used by any of the test steps that are actually in the feature files.